modules/swagger-codegen/src/main/resources/php/api.mustache

```diff
@@ -237,22 +248,14 @@ use {{invokerPackage}}\ObjectSerializer;
                 }
                 $httpBody = new MultipartStream($multipartContents); // for HTTP post (form)
 
+            } elseif ($headers['Content-Type'] === 'application/json') {
```

Comment:
@baartosz do you mind adding a new method (e.g. isJsonMime) to determine if a MIME string is "JSON"?

For Java, we've https://github.com/swagger-api/swagger-codegen/blob/master/samples/client/petstore/java/okhttp-gson/src/main/java/io/swagger/client/ApiClient.java#L752-L754

and here are some tests:

https://github.com/swagger-api/swagger-codegen/blob/fc01457d77315afdd023c45587d6f48a26a2d131/samples/client/petstore/java/okhttp-gson/src/test/java/io/swagger/client/ApiClientTest.java#L81-L102